ejp-rd-vp/vp-dp-frontend
# 80 add connect page

**What's in the PR
* Add the connect page

**How to test manually
* Click the Connect button on the home page
* You should be linked to the connect page
